doc: add missing parenthesis in TLSSocket section (#44512)
nodejs/node: doc/api/tls.md

Instances of `tls.TLSSocket` implement the duplex [Stream][] interface.

> Methods that return TLS connection metadata (e.g.

Reviewer: Totally optional, but if you want to get rid of this `e.g.` too, that would be 👍 by me (and the [Microsoft style guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)).

```suggestion
Methods that return TLS connection metadata (such as
```
